package.json

```diff
@@ -170,7 +173,7 @@
     "@types/node": "^10.12.21",
     "@types/node-fetch": "^2.1.7",
     "@types/sinon": "^7.0.7",
-    "@types/tmi.js": "~1.4.0",
+    "@types/tmi.js": "~1.4.2",
```

Comment:
I'm not sure we even need these types anymore since we are using Comfy. However, that might be outside the scope of this PR.